data_safe_haven/README.md

```diff
@@ -14,6 +14,14 @@ Install the following requirements before starting
 > dsh context create
```

Comment:
```suggestion
> dsh context create
> dsh context switch ...
```

Creating a context doesn't automatically select that context, so a switch is necessary here.

Reply:
Yes, although I think it is `switch` after `add` but before `create`. `create` deploys the infrastructure. I've updated the suggestion.

We could also change the behaviour of `dsh context create` so that when you make a new context you automatically switch. You could open an issue if you think there is a better way to do this.